worlds/shivers/data/locations.json

```
@@ -320,6 +311,33 @@
     "Skull Dial Bridge": [
       "Accessible: Storage: Skull Bridge",
       "Puzzle Solved Skull Dial Door"
+    ],
+    "Water Capture": [
+      "Ixupi Captured Water"
+    ],
+    "Wax Capture": [
+      "Ixupi Captured Wax"
+    ],
+    "Ash Capture": [
+      "Ixupi Captured Ash"
+    ],
+    "Oil Capture": [
+      "Ixupi Captured Oil"
+    ],
+    "Cloth Capture": [
+      "Ixupi Captured Cloth"
+    ],
+    "Wood Capture": [
+      "Ixupi Captured Wood"
+    ],
+    "Crystal Capture": [
+      "Ixupi Captured Crystal"
+    ],
+    "Sand Capture": [
+      "Ixupi Captured Sand"
+    ],
+    "Metal Capture": [
+      "Ixupi Captured Metal"
     ]
   }
-}   
+}
```

Newline at end of file
```suggestion
}

```

---

I don't know what the benefit is to having a new line at the end of the file but I have added it.